### DISCUSSION TITLE
Mojo 🔥 Standard Library is now Open Source! (plus ⚡️ MAX 24.2!)

### DISCUSSION THREAD

**Discussion by hogepodge** [modular/modular#2019] [General]
📣 Exciting news! We’ve open sourced the Mojo standard library core modules, and are now shipping nightly builds to support community development! [Read more about this release, and how this fits into our vision for Mojo’s open source future](https://modul.ar/open-source-blog).

[MAX⚡ and Mojo🔥 24.2 are also live!](https://www.modular.com/blog/max-24-2-is-here-whats-new) In this latest release, [MAX Engine 🏎️ now support TorchScript dynamic input tensor shapes](https://www.modular.com/blog/leveraging-max-engines-dynamic-shape-capabilities), and Mojo 🔥 now supports keyword arguments to Python methods. Read more in the changelogs: ⬇️

- [MAX changelog and update steps.](https://modul.ar/max-changelog)
- [Mojo changelog and update steps.](https://modul.ar/mojo-changelog)

**Comment by idaix** [comment]
Here we go 🔥

**Comment by matt2000** [comment]
Is the compiler open-source? If not, then I still can't use Mojo. 😥

**Reply by hogepodge** [reply]
Hi @matt2000, the Mojo compiler isn't open source because it's still under heavy development, and we want to prioritize the language development. We understand that this makes Mojo a non-starter for some use cases, but our plan is to progressively open source Mojo as development stabilizes.
